inscriptions/ISic000264.xml

```diff
@@ -187,51 +192,22 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR127546" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#TNS">
                 <ab>
-                    <lb n="1"/>
-                    <expan>
-                        <abbr>P</abbr>
-                        <ex>ublius</ex>
-                    </expan>
-                    Cor
-                    <unclear>n</unclear>
-                    <supplied reason="lost">elius</supplied>
-                    <lb n="2"/>Au
-                    <gap reason="lost" extent="unknown" unit="character"/>
-                    <lb n="3"/>
-                    <expan>
-                        <abbr>vix</abbr>
-                        <ex>it</ex>
-                    </expan>
-                    an
-                    <supplied reason="lost">
-                        <expan>
-                            <abbr>n</abbr>
-                            <ex>is</ex>
-                        </expan>
-                        ---
-                    </supplied>
-                    <lb n="4"/>dié
-                    <expan>
-                        <abbr>b</abbr>
-                        <ex>us</ex>
-                    </expan>
-                    XX
-                    <unclear>V</unclear>
-                    <gap reason="lost" extent="unknown" unit="character"/>.
+                  <lb n="1"/><name><expan><abbr>P</abbr><ex>ublius</ex></expan></name> <g ref="interpunct">·</g> <name>Cor<unclear>n</unclear><supplied reason="lost">elius</supplied></name>
+                  <lb n="2"/><name>Au<gap reason="lost" extent="unknown"></gap></name>
+                  <lb n="3"/><expan><abbr>vix</abbr><ex>it</ex></expan> <g ref="interpunct">·</g> an<supplied reason="lost">nis</supplied>
+                  <lb n="4"/><expan><abbr>dieb</abbr><ex>us</ex></expan> <g ref="interpunct">·</g> <num value="25">XXV</num>
```

Comment:
as per discussion, change value to atLeast="25" atMost="29", add a gap inside the num, and potetnially add a gap on the end of each of the preceding lines also.